src/Simplex/Chat/Migrations/M20230317_hidden_profiles.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Migrations.M20230317_hidden_profiles where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20230317_hidden_profiles :: Query
+m20230317_hidden_profiles =
+  [sql|
+PRAGMA ignore_check_constraints=ON;
+
+ALTER TABLE users ADD COLUMN view_pwd_hash BLOB NULL;
+ALTER TABLE users ADD COLUMN view_pwd_salt BLOB NULL;
+ALTER TABLE users ADD COLUMN show_ntfs INTEGER NOT NULL DEFAULT 1;
```

Comment:
does it complete successfully on existing database? I don't remember exactly why we started doing CHECK and not just NOT NULL in such cases, maybe it was failing on some platforms/sqlite versions?

Reply:
will test on android, removed disabling checks/update